.github/workflows/lint.yml

```diff
@@ -44,6 +65,11 @@ jobs:
       contents: read
     runs-on: ubuntu-latest
     steps:
+      - uses: step-security/harden-runner@main
+        with:
+          allowed-endpoints:
+            agent.api.stepsecurity.io:443
```

Comment:
why is this one here? shouldn't that be automatic by using the harden-runner?

Reply:
I wanted it to be explicit. In fact, because of that, I got some feedback/ concern about data being sent to agent.api.stepsecurity.io. The agent code is open source, and it is only sending metadata about network and processes. Do you have ideas on how to mitigate concern that users may have about sending metadata to that API?

Reply:
I don't see why anyone would have that concern in the first place in an open source workflow.

Reply:
tbh I don't see why anyone would have that concern in the first place in an open source workflow.

Reply:
It was only because this is a new service, not a fundamental issue. Still on fence on whether agent.api.stepsecurity.io should be listed explicitly in the allowed-endpoints or just be implicit...

Reply:
Removed agent.api.stepsecurity.io as explicit endpoint. It is added implicitly now.